gradle/libs.versions.toml

```diff
@@ -1,27 +1,24 @@
 [versions]
 
-gradlePlugin-kotlin = "1.9.22"
+gradlePlugin-kotlin = "2.0.0-Beta4"
 # See: https://kotlinlang.org/docs/gradle-configure-project.html#apply-the-plugin
-gradlePlugin-android = "4.2.2"
-gradlePlugin-android-dokkatoo = "8.0.2"
+gradlePlugin-android = "7.1.3"
```

Comment:
> gradlePlugin-android = "4.2.2"
> gradlePlugin-android-dokkatoo = "8.0.2"

These are unused at the moment, so I've removed them (and the dependencies that depend on them) to reduce noise. Can be added once it's needed

> gradlePlugin-android = "7.1.3"
KGP 2.0.0 now requires AGP 7.1.3+, so I had to bump it as well